packages/utilities/psammead-assets/src/svgs/archive.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const archive = {
+  group: (
+    <g
+      fillRule="evenodd"
+      stroke="#fff"
```

Comment:
Changing this to black ensures that without CSS, the logo renders correctly, with a black stroke. This matches implementation in our other Brand SVGs. 

```suggestion
      stroke="#000"
```

After the change, looking at http://localhost:8180/iframe.html?id=utilities-svgs-brand-svgs--archive with CSS disabled, gives us this logo: 
<img width="230" alt="Screenshot 2019-11-26 at 10 17 25" src="https://user-images.githubusercontent.com/3028997/69621073-56dfb000-1036-11ea-9918-c859e4629ef0.png">